modular_citadel/code/modules/eventmaps/Spookystation/JTGSZwork.dm

```diff
@@ -0,0 +1,1122 @@
+	has_gravity = STANDARD_GRAVITY //We have gravity
+	icon = 'modular_citadel/code/modules/eventmaps/Spookystation/areas.dmi' //It unsets the icon. don't make a err icon.
+	requires_power = 0 // We don't need power anywhere.
+	flags_1 = NONE
+	dynamic_lighting = DYNAMIC_LIGHTING_FORCED
+
+/area/eventmap/outside //We are outside
+	name = "Outside"
+	icon_state = "outside"
+	outdoors = 0 //I set outdoors to false. So areas can be edited.
+
+/area/eventmap/inside //We are inside, all things are pretty normal.
+	name = "Inside"
+	icon_state = "inside"
+
+/area/eventmap/mountain //Mostly so I can see the area lines of the mountain area in the minimap.
+	name = "Mountain"
+	icon_state = "mountain"
+	var/mountain = 1
+
+/area/eventmap/mountaininside
+	name = "Silent Mountain Inside"
+	icon_state = "mountain_inside"
+	outdoors = 0
+
+/*
+	OUTSIDE WALLS I WANT NOT NEED
+										*/
+//These exist mostly to limit the amount of space we use organically really.
+//Decided to just use the denserock within the regular code.
+
+/turf/closed/indestructible/spookytime/matrixblocker //Two times the reference power.
+	name = "matrix"
+	icon = 'modular_citadel/code/modules/eventmaps/Spookystation/iconfile32.dmi'
+	icon_state = "matrix"
+	desc = "<font color='#157206'>You suddenly realize the truth - there is no spoon.<br>Digital simulation ends here ONCE AGAIN.</font>"
+
+/*
+	OUTSIDE TURFS WITH NO GEN JUS MIDNIGHT LIGHT BABY
+														*/
+
+//In a ideal world, we would have split the turfs onto a single parent.
+//Then we would tree from INSIDE and OUTSIDE, with outside having the lighting set, for the day/night subsystem to change.
+//Outside would also have the planetary atmos and config on it.
+//Inside would be a case to case basis depending on if you want it to scrub or not.. and not have the lighting.
+//Along with what needs to be constructed on etc.
+//This is not a ideal world so enjoy the overload.
+
+//Parent of all our outside turfs. Both the inside and outside should be on a parent like this.
+/turf/open/floor/spooktime //But for now, we just handle what is outside, for light control etc.
+	name = "You fucked up pal"
+	desc = "Don't use this turf its a parent and just a holder."
+	planetary_atmos = 1 //REVERT TO INITIAL AIR GASMIX OVER TIME WITH LINDA. AKA SUPERSCRUBBER
+	light_range = 3 //MIDNIGHT BLUE
+	light_power = 0.15 //NOT PITCH BLACK, JUST REALLY DARK
+	light_color = "#00111a" //The light can technically cycle on a timer worldwide, but no daynight cycle.
+	baseturfs = /turf/open/floor/plating/spookbase/dirtattachmentpoint //If we explode or die somehow, we just become grass
+	gender = PLURAL //THE GENDER IS PLURAL
+	tiled_dirt = 0 //NO TILESMOOTHING DIRT/DIRT SPAWNS OR SOME SHIT
+
+/turf/open/floor/spooktime/pry_tile(obj/item/I, mob/user, silent = FALSE)
+	return //No prying these tiles, you instead shovel it if avail.
+
+/*
+	Baseturf, when we call scrapeaway() after a shoveling. So people can attach tiles
+																						*/
```

Comment:
to prevent issues with these tiles missing broken sprites. Tiny bit hacky but who cares noone needs to bomb or incinerate tiles during a ball (feel free to basically write this in your editor yourself because mine might have fucky lineendings)
```suggestion

/turf/open/floor/spooktime/break_tile()
	return
/turf/open/floor/spooktime/burn_tile()
	return
																						*/
```